packages/icons/package.json

```diff
@@ -28,7 +28,7 @@
     "build",
     "CHANGELOG.md"
```

Comment:
irrelevant

Reply:
What about those changes do you see as irrelevant? I have changed the author (as defined above), otherwise made sure that the metadata contains the correct urls? This is for example needed for links at npmjs.com to work properly.